pages/design-tokens/typesetting/font-family.md

```diff
@@ -173,20 +175,19 @@ $theme-font-type-sans:   'source-sans-pro';
 $theme-font-type-serif:  'merriweather';
 ```
 
-**Then use the type variables you just set to set the [role](#role-based-tokens){:.token} family tokens.** Set any unused types to `false`.
+**Then use the [type](#type-based-tokens){:.token} tokens you just set to define the [$theme-font-role-]({{ site.baseurl }}/documentation/settings/#typography-settings) settings variables.** These settings define the value of the [role](#role-based-tokens){:.token} family tokens. Set any unused types to `false`.
 
 ```sass
-$theme-font-role-ui:       $theme-font-sans;
-$theme-font-role-heading:  $theme-font-serif;
-$theme-font-role-body:     $theme-font-sans;
-$theme-font-role-code:     $theme-font-mono;
-$theme-font-role-alt:      $theme-font-serif;
+$theme-font-role-ui:       'sans';
+$theme-font-role-heading:  'serif';
+$theme-font-role-body:     'sans';
+$theme-font-role-code:     'mono';
+$theme-font-role-alt:      'serif';
```

Comment:
Replaced settings variable values with tokens since using the settings variables causes a compilation error.